website/docs/docs/dbt-versions/experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Preview new and experimental features in dbt Cloud"
+id: "experimental-features"
+sidebar_label: "Preview new dbt Cloud features"
+description: "Gain early access to many new dbt Labs experimental features by enabling this in your profile."
+---
+
+dbt Labs often beta tests features before deciding to continue on the [Product lifecycle](https://docs.getdbt.com/docs/dbt-versions/product-lifecycles#dbt-cloud).
+
+You can access experimental features if you want to preview beta features that haven’t yet been released to dbt Cloud. You can toggle on or off all experimental beta features in one place. As with other beta features, these features:
+
+- May not be feature-complete or fully stable as we’re actively developing them.
+- Could be discontinued at any time.
+- May require feedback from you to understand their limitations or impact. Each experimental feature collects feedback directly in dbt Cloud.
+- May have limited technical support and be excluded from our Support SLAs.
+- May not have public documentation available.
+
+To enable or disable experimental features:
+
+1. Navigate to Your Profile settings by clicking the gear icon in the top right.
+2. Find Experimental features at the bottom of the page.
+3. Click **Beta** to toggle the features on or off as shown in the following image.
```

Comment:
Is this going to be the only way to access ANY beta feature in dbt Cloud moving forward? If it's betas related to only particular areas we may need to call that out.

Reply:
@matthewshaver  I think will still be using feature flags, but you bring up a good potential point of confusion.  

@GideonShils Will this be the way to opt-in to all beta features with this 1st iteration? and could they be in all areas of the product?

Reply:
Not all features will necessarily go through the experimental features flow. This is a new space for us, so the details on how exactly this will be used aren't fully worked out quite yet 😄.  For some features, we may do a more "private" beta where we reach out to customers directly, but this opens up the _option_ for shipping to a more "opted in" set of users instead. It's up to the devs / product owners how they want to ship their specific feature.